pkg/volume/host_path/host_path_test.go

```diff
@@ -66,8 +69,10 @@ func TestGetAccessModes(t *testing.T) {
 }
 
 func TestRecycler(t *testing.T) {
+	tmpDir, cleanUp := maketmpdir.MakeTmpDir(t, "hostpathTest")
+	defer cleanUp()
```

Comment:
The extra level of indirection seems unnecessary here. Why not just call `defer os.RemoveAll(tmpDir)` instead of having `MakeTmpDir(...)` return a cleanup method--unless you anticipate the cleanup method to do something additional?